_toc.yml

```diff
@@ -231,79 +233,87 @@ entries:
           entries:
             - glob: docs/tools/cli/user/*
         - file: docs/tools/cli/vpc
+      
+      # -------- API --------
       - file: docs/tools/api
         title: Aiven API
         entries:
           - file: docs/tools/api/examples
+      
+      # -------- TERRAFORM --------
       - file: docs/tools/terraform
-        title: Aiven Terraform provider
+        title: Aiven Provider for Terraform
```

Comment:
I can see you're using "Aiven Provider for Terraform" consistently. I like "Aiven Terraform Provider" better but don't have any strong argument, except it's just a bit shorter. Obv, this is your call. Also, I can see you capitalize "provider", meaning you treat the whole phrase as a proper name. It seems there are quite a few instances of this phrase elsewhere in the docs using the lower case. It'd be good to also update them so that we use consistently the variant you chose (Aiven Provider for Terraform).